src/components/Settings/CoreSettings.tsx

```diff
@@ -217,9 +217,12 @@ export const CoreSettings = () => {
         commands. Make sure you trust all users you give these permissions to
       </ConfirmDialog>
       {openPortModal}
-      <div className="flex w-full flex-col gap-4 @4xl:flex-row">
-        <div className="w-[28rem]">
-          <h2 className="text-h2 font-bold tracking-medium"> Core Settings </h2>
+
+      <div className="flex w-full flex-col gap-4 @4xl:flex-row px-6">
```

Comment:
It might be better to add a max-width on this page, just like the dashboard pages.